Dfe.Academies.Academisation.Domain.Core/ProjectAggregate/ProjectDetails.cs

```diff
@@ -39,7 +39,7 @@ public class ProjectDetails : IEquatable<ProjectDetails>
 	public string? NameOfTrust { get; init; }
 	public string? SponsorReferenceNumber { get; init; }
 	public string? SponsorName { get; init; }
-	public string? AcademyTypeAndRoute { get; init; }
+	public string? AcademyTypeAndRoute { get; set; }
```

Comment:
probably needs to be init or private, I had to create backing fields for the trust name and reference to maintain init functionality but also stop it from being externally editable without going through the aggregate